CMakeLists.txt

```diff
@@ -181,6 +181,7 @@ SET(${PROJECT_NAME}_PYTHON_HEADERS
   python/algorithms.hpp
   python/parsers.hpp
   python/explog.hpp
+  multibody/parser/python.hpp
```

Comment:
This line must appear in ${PROJECT_NAME}_MULTIBODY_PARSER_HEADERS if ${BUILD_PYTHON_INTERFACE}